doc/sphinx/addendum/ring.rst

```diff
@@ -308,7 +308,7 @@ following property:
 
 The syntax for adding a new ring is 
 
-.. cmd:: Add Ring @ident : @one_term {? ( {+, @ring_mod } ) }
+.. cmd:: Add Ring @ident @open_binders : @one_term {? ( {+, @ring_mod } ) }
```

Comment:
Please add some explanation here what the open_binders does.

Reply:
I added a line. Is it good enough to understand their rôle?

Incidentally, it would be more consistent to use a `:=` than a `:` since this is a definition...

Reply:
Yes, good enough, thanks.